src/Shamsheer.Messenger.Api/Controllers/ChannelAssets/ChannelAssetsController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Shamsheer.Service.Configurations;
+using Shamsheer.Service.Interfaces.ChannelAssets;
+
+namespace Shamsheer.Messenger.Api.Controllers.ChannelAssets;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ChannelAssetsController : ControllerBase
+{
+    IChannelAssetService _channelAssetService;
+
+    public ChannelAssetsController(IChannelAssetService channelAssetService)
+    {
+        _channelAssetService = channelAssetService;
+    }
+
+
+    [HttpPost]
+    public async Task<IActionResult>PostAsync(IFormFile file)
+        =>Ok( await _channelAssetService.CreateAsync(file));
+
+
+    [HttpGet("{channelId}")]
```

Comment:
shunda {channel-id} deb yozgan ma'qul, naming convention'ga mos bo'ladi.

Reply:
etibor bermapman to'g'irlab qo'yaman